### ISSUE TITLE
Matplotlib backend issue on Windows

### ISSUE THREAD

**Issue by thusithathilina** [autonomio/talos#182]
Thanks so much for coming here to raise an issue. Please take a moment to 'check' the below boxes:

- [x ] I'm up-to-date with the latest release:
    
      pip install -U talos

- [ ] I've confirmed that my Keras model works outside of Talos.

If you still have an error, please submit **complete trace** and a code with: 

- output of shape for x and y e.g. (212,12)
- Talos params dictionary
- The Keras model wired for Talos
- Description of extra variables in the model

You can provide the code in pastebin / gist or any other format you like. 
I've following code segment. https://pastebin.com/QCaXGQt0 when I'm running the code through the python console. It get exit without any errors.

Output
PS C:\Work\PycharmProjects\Predictor\detection> python3.exe .\nn_with_param_optimizer.py -V
C:\Python36\lib\site-packages\chances\plots.py:7: UserWarning:
This call to matplotlib.use() has no effect because the backend has already
been chosen; matplotlib.use() must be called *before* pylab, matplotlib.pyplot,
or matplotlib.backends is imported for the first time.

The backend was *originally* set to 'TkAgg' by the following code:
  File ".\nn_with_param_optimizer.py", line 4, in <module>
    import talos as ta
  File "C:\Python36\lib\site-packages\talos\__init__.py", line 2, in <module>
    from .scan.Scan import Scan
  File "C:\Python36\lib\site-packages\talos\scan\Scan.py", line 1, in <module>
    from .scan_prepare import scan_prepare
  File "C:\Python36\lib\site-packages\talos\scan\scan_prepare.py", line 3, in <module>
    from ..utils.validation_split import validation_split
  File "C:\Python36\lib\site-packages\talos\utils\validation_split.py", line 2, in <module>
    from wrangle import shuffle
  File "C:\Python36\lib\site-packages\wrangle\__init__.py", line 24, in <module>
    from .ols_corr import ols_corr
  File "C:\Python36\lib\site-packages\wrangle\ols_corr.py", line 3, in <module>
    import statsmodels.api as sm
  File "C:\Python36\lib\site-packages\statsmodels\api.py", line 40, in <module>
    from .multivariate.factor import Factor
  File "C:\Python36\lib\site-packages\statsmodels\multivariate\factor.py", line 15, in <module>
    import matplotlib.pyplot
  File "C:\Python36\lib\site-packages\matplotlib\pyplot.py", line 71, in <module>
    from matplotlib.backends import pylab_setup
  File "C:\Python36\lib\site-packages\matplotlib\backends\__init__.py", l

-------------------------------------------------------------------------

**Comment by mikkokotila** [comment]
This is an upstream problem related with matplotlib. Can you first try upgrading to latest version of Talos: 

`pip install git+https://github.com/autonomio/talos@daily-dev`

...and see what happens.

**Comment by thusithathilina** [comment]
@mikkokotila Thanks for the suggestion. I tried your suggestion. But still the same result :(
Process get exit without any errors.

```
PS C:\Work\PycharmProjects\Predictor\detection> python3.exe .\nn_with_param_optimizer.py -v
Using TensorFlow backend.
C:\Python36\lib\site-packages\chances\plots.py:7: UserWarning:
This call to matplotlib.use() has no effect because the backend has already
been chosen; matplotlib.use() must be called *before* pylab, matplotlib.pyplot,
or matplotlib.backends is imported for the first time.

The backend was *originally* set to 'TkAgg' by the following code:
  File ".\nn_with_param_optimizer.py", line 5, in <module>
    import talos as ta
  File "C:\Python36\lib\site-packages\talos\__init__.py", line 2, in <module>
    from .scan.Scan import Scan
  File "C:\Python36\lib\site-packages\talos\scan\Scan.py", line 1, in <module>
    from .scan_prepare import scan_prepare
  File "C:\Python36\lib\site-packages\talos\scan\scan_prepare.py", line 3, in <module>
    from ..utils.validation_split import validation_split
  File "C:\Python36\lib\site-packages\talos\utils\validation_split.py", line 2, in <module>
    from wrangle import shuffle
  File "C:\Python36\lib\site-packages\wrangle\__init__.py", line 24, in <module>
    from .ols_corr import ols_corr
  File "C:\Python36\lib\site-packages\wrangle\ols_corr.py", line 3, in <module>
    import statsmodels.api as sm
  File "C:\Python36\lib\site-packages\statsmodels\api.py", line 40, in <module>
    from .multivariate.factor import Factor
  File "C:\Python36\lib\site-packages\statsmodels\multivariate\factor.py", line 15, in <module>
    import matplotlib.pyplot
  File "C:\Python36\lib\site-packages\matplotlib\pyplot.py", line 71, in <module>
    from matplotlib.backends import pylab_setup
  File "C:\Python36\lib\site-packages\matplotlib\backends\__init__.py", line 16, in <module>
    line for line in traceback.format_stack()

  matplotlib.use('Agg')
  0%|                                                                                                                                                 | 2/28800 [04:23<1076:24:16, 134.56s/it] PS C:\Work\PycharmProjects\Predictor\detection>
```

**Comment by thusithathilina** [comment]
@mikkokotila  Just to update. When I tried using PyCharm. There I can see the following exit code
`Process finished with exit code -1073741819 (0xC0000005)
`

**Comment by blu3r4y** [comment]
I experienced the same error code (0xC0000005) on Windows, but with [minimal_talos_example.py](https://gist.github.com/mikkokotila/4c0d6298ff0a22dc561fb387a1b4b0bb).

Upgrading Tensorflow from 1.8 to 1.14 solved the issue for me.

**Comment by mikkokotila** [comment]
@blu3r4y Nice, thanks so much for positing the solution. Will close here as it seems to have a working resolution.
